packages/cargo/src/types/types-v2.ts

```diff
@@ -71,9 +71,18 @@ export interface ScheduledReportListV2 {
   schedules: ScheduledReportV2[];
 }
 
+export interface ReportAttachment {
+    name: string;
+    url: string;
+}
+
 export interface ReportArtifactV2 {
   result_id?: string;
-  is_rerference?: boolean;
+  is_reference?: boolean;
+  create_time?: number;
+  metadata?: {[key:string]: unknown};
+  result_count?: number;
+  attachments?: ReportAttachment[];
```

Comment:
Did you know you can use inline interfacing for simple objects like this?

```
export interface ReportArtifactV2 {
  result_id?: string;
  is_rerference?: boolean;
  is_reference?: boolean;
  create_time?: number;
  metadata?: {[key:string]: unknown};
  result_count?: number;
  attachments?: {
    name:string,
    url:string
  }[];
}

Reply:
No, good to know! Thanks for your wisdom, Mr. Nielsen!